src/main.py

```diff
@@ -415,7 +400,6 @@ def process_response(response):
 			for directive in j['messageBody']['directives']:
 				if directive['namespace'] == 'SpeechSynthesizer':
 					if directive['name'] == 'speak':
-						platform.indicate_recording(False)
```

Comment:
it might be helpful for review if this one line change had more explanation in the git commit message and possibly resided in it's own commit. It looks like this code line was removed in 82a97187186b2e203779ae719b0c076099d06207 and then came back in 2374632e0da13cc480aaf3887e970d5dca597573 (likely because of a rebase from another branch). This is not uncommon and  understandable but hard to review without a little more info